src/jitstate.rs

```diff
@@ -520,6 +554,25 @@ impl<'a> JitState<'a> {
     jit_reexport!(retval_s, rv: Reg);
     jit_reexport!(retval_us, rv: Reg);
     jit_reexport!(retval_i, rv: Reg);
+
+    pub fn get_note(
+        &self,
+        code: JitPointer,
+        name: Option<&mut * mut std::os::raw::c_char>,
+        file: Option<&mut * mut std::os::raw::c_char>,
```

Comment:
I could return a `CStr` here, potentially, but I am for now trying to stay as thin as possible.